### PR TITLE
Missing content ghi145

### PR DESCRIPTION
**Pull request by ejzn**
This fixes issue #145. Config.xml files with missing missing content elements where being passed through. Now an error is displayed to notify the developer that a content element is required.

### CODE REVIEW

**Comment by tracyli** [comment]
tested in windows and mac with http://ci0000003863287:8080/hudson/job/BB10-Webworks-Packager-missing-content-GHI145/4/ and this issue has been fixed: error 'Please enter a valid content element' shows up.

File: test/unit/lib/config-parser.js [hunk 1]
```diff
@@ -171,6 +171,14 @@ describe("config parser", function () {
             configParser.parse(configPath, session, {});
         }).toThrow(localize.translate("EXCEPTION_INVALID_ID"));
     });
+
+    it("Fails when missing content error is not shown", function () {
+        var missingContentConfigPath = path.resolve("test/config-missing-content.xml");
```
Comment (jamesjhedley) [inline test/unit/lib/config-parser.js]: This should be mocked and the "config-missing-content.xml" file removed.

File: lib/localize.js [hunk 2]
```diff
@@ -82,6 +82,9 @@ var Localize = require("localize"),
         "EXCEPTION_INVALID_ID": {
             "en": "Please enter a valid application id"
         },
+        "EXCEPTION_INVALID_CONTENT": {
+            "en": "Please enter a valid content element"
```
Comment (jamesjhedley) [inline lib/localize.js]: @nukulb @kwallis Do we want to use the same error message from Snarf? (see below).

```
[ERROR] Invalid config.xml - failed to parse the <content> element(Invalid source or the source is not specified.)
```

Comment (jamesjhedley) [inline lib/localize.js]: Lets go ahead and update this.

**Comment by jamesjhedley** [comment]
Code review complete.

**Comment by nukulb** [comment]
merged manually

**Pull request closed without merging**